Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+pipeline {
+    agent { label 'centos7-docker-8c-8g' }
+    environment {
+        SECURITY_SERVICE_NEEDED = false
+    }
+    stages {
+        stage('Prep') {
+            steps {
+                script {
+                    sh 'curl -v https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/nightly-build/compose-files/docker-compose-nexus.yml > docker-compose.yml'
+                    docker
+                        .image('nexus3.edgexfoundry.org:10003/edgex-devops/edgex-compose:latest')
+                        .inside('-u 0:0 --entrypoint= --net host --security-opt label:disable -v /var/run/docker.sock:/var/run/docker.sock')
+                    {
+                        sh 'docker ps -a'
+                        sh 'docker-compose up -d'
+                        sh 'sleep 60'
+                        sh 'docker ps -a'
+                    }
+                }
+            }
+        }
+
+        stage('CIS Benchmark') {
+            steps {
+                sh 'docker run --net host --pid host --cap-add audit_control -e DOCKER_CONTENT_TRUST=$DOCKER_CONTENT_TRUST -v /var/lib:/var/lib -v /var/run/docker.sock:/var/run/docker.sock -v /usr/lib/systemd:/usr/lib/systemd -v /etc:/etc --label docker_bench_security docker/docker-bench-security -c check_4_1,check_5_3,check_5_4,check_5_5,check_5_6,check_5_7,check_5_9,check_5_12,check_5_15,check_5_16,check_5_19,check_5_20,check_5_21,check_5_24,check_5_25,check_5_29,check_5_30,check_5_31'
```

Comment:
Can put this on multiple lines? It's a bit hard to read this whole command.

Maybe do something like:
```Groovy
def checks = [
    'check_4_1',
    'check_5_3',
    'check_5_4',
    'check_5_5',
    'check_5_6',
    'check_5_7',...
]
sh "docker run...  -c ${checks.join(',')}"